docs/build/apps/wallet/sep7.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: URI Scheme to facilitate delegated signing
+sidebar_position: 80
+---
+
+import Header from "./component/header.mdx";
+import { WalletCodeExample as CodeExample } from "@site/src/components/WalletCodeExample";
+
+<Header />
+
+The [Sep-7] standard defines a way for a non-wallet application to construct a URI scheme that represents a specific transaction for an account to sign. The scheme used is `web+stellar`, followed by a colon.
+Example: `web+stellar:<operation>?<param1>=<value1>&<param2>=<value2>`
+
+## Tx Operation
+
+The tx operation represents a request to sign a specific transaction envelope, with [some configurable parameters].
+
+<CodeExample>
+```typescript
+const sourceAccountKeyPair = "G..."
+const destinationAccountKeyPair = "G...
+const txBuilder = await stellar.transaction({
+  sourceAddress: sourceAccountKeyPair,
+});
+const tx = txBuilder.createAccount(destinationAccountKeyPair).build();
+const xdr = encodeURIComponent(tx.toEnvelope().toXDR().toString("base64"));
+const callback = encodeURIComponent("https://example.com");
+const txUri = `web+stellar:tx?xdr=${xdr}&callback=${callback}`;
+const uri = new Sep7Tx(txUri);
+// uri can be parsed and transaction can be signed/submitted by an application that implements Sep-7
+```
+</CodeExample>
+
+You can set replacements to be made in the xdr for specific fields by the application, these will be added in the [Sep-11 transaction representation format] to the URI.
+
+<CodeExample>
+```typescript
+const uri = new Sep7Tx(txUri);
+uri.addReplacement({
+  id: "X",
+  path: "sourceAccount",
+  hint: "account from where you want to pay fees",
+});
+```
+</CodeExample>
+
+## Pay Operation
+
+The pay operation represents a request to pay a specific address with a specific asset, regardless of the source asset used by the payer. You can [configure parameters] to build the payment operation.
+
+<CodeExample>
+```typescript
+const destination = "G...";
+const amount = "120.1234567";
+const memo = "memo";
+const message = encodeURIComponent("pay me with lumens");
+const originDomain = "example.com";
+const payUri = `web+stellar:pay?destination=${destination}&amount=${amount}&memo=${memo}&msg=${message}&origin_domain=${originDomain}`;
+const uri = new Sep7Pay(payUri);
```

Comment:
`const uri: Sep7Pay = parseSep7Uri(payUri);`

Reply:
Added in 566202ca83fc41934a712e196afa6af3f81cd2cd